### PR TITLE
Add new header, incorporate use of 'vocabulary'

### LINKED ISSUE #70: [Bug] clicking on whole header opens https://creativecommons.org/
## Description
The whole header div is enclosed inside the anchor tag due to which the whole header act as link but technically only the logo should act as link.

Also according to UX, clicking the logo should redirect the home page but here it is opening the CreativeCommons.org

## Screenshots

before image
image

image

1. first remove logo from style.css and add img tag to html
2. put anchor tag across the image 

after changing the code 

image

## Environment

 - Device: (_eg._ iPhone Xs; laptop)
 - OS: (_eg._ iOS 13.5; Fedora 32)
 - Browser: (_eg._ Safari; Firefox)
 - Version: (_eg._ 13; 73)
 - Other info: (_eg._ display resolution, ease-of-access settings)

## Resolution

- [x] I would be interested in resolving this bug.
- [x]  removing whole header as link making only the logo as link
- [x] redirecting it to the home page and not to https://creativecommons.org/

### LINKED ISSUE #210: [Feature] Logo should include site name per Vocabuarly design system
## Problem

1. ~~Clicking on the current logo navigates to the home page of [creativecommons.org](https://creativecommons.org/) instead of navigating to the dedicated [resources.creativecommons.org]( https://resources.creativecommons.org/)'s home page.~~
   - duplicates #70 
   - resolved by #145 
3.  The current logo does not include the site's name like other primary CC websites' logo.

## Description

The proposed feature involves updating the logo name to include 'Resources,' aligning it with other primary CC websites' logos such as [open source](https://opensource.creativecommons.org/), [global summit](https://summit.creativecommons.org/), and [global network](https://network.creativecommons.org/). This change would enhance user experience by providing direct access to CC's resources page

## ScreenShots
[**resources.creativecommons.org**](https://resources.creativecommons.org/)
resources archive

[**opensource.creativecommons.org**](https://opensource.creativecommons.org/)
opensource

[**summit.creativecommons.org**](https://summit.creativecommons.org/)
global summit

[**network.creativecommons.org**](https://network.creativecommons.org/)
global networks

## Implementation

- [x] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by Sana-Maryam90**
## Fixes

Fixes #210 by @Sana-Maryam90 and fixes #70 by @RishabhRajJoshi
- #70 
- #210

## Description

The CreativeCommons logo now navigates to the Resources' home page [**resources.creativecommons.org**](https://resources.creativecommons.org/) instead of navigating to [**creativecommons.org**](https://creativecommons.org/). Also, by using vocabulary, logo styles have been changed similar to [**creativecommons.org**](https://creativecommons.org/) and other primary CC websites. Additionally, the issue with logo behavior #70, which previously allowed navigation when clicking anywhere along the header, has been addressed.

## Technical details

The URL in the anchor tag was changed. The logo styles were updated by utilizing vocabulary in style.css

## Video Demonstration

It is visible in the video that clicking on logo navigates to Resources Home page rather than [**creativecommons.org**](https://creativecommons.org/). Also, we can see that mouse hover on header area is not clickable anymore resolving #70.
https://github.com/creativecommons/cc-resource-archive/assets/99211506/f1b71770-2860-43c8-97c9-b8ea63bf593c

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by possumbilities** [comment]
Hi! @Sana-Maryam90 

Changes can't be made directly to things within the `/vocabulary` directory. If changes are needed they'll need to be overridden styles elsewhere. I see you altered the `vocabulary.css`. If changes are needed they need to be within the `style.css` file or elsewhere rather than here. That way future updates to Vocabulary don't have to contend with accidentally overriding localized changes.

I also don't think you need the main title anymore since the new product logo should serve as the new page title.

**Comment by Sana-Maryam90** [comment]
I'll correct it as soon as possible

**Comment by Sana-Maryam90** [comment]
Hi @possumbilities 

All changes I made previously in vocabulary.css have been reverted. Also, the main title has been removed as you suggested.

main title

File: docs/_includes/header.html [hunk 1]
```diff
@@ -1,42 +1,46 @@
...
     </div>
-</div>
-
-<!-- Main Content -->
-
-<a href="//creativecommons.org/">
-    <div id="bannercc"></div>
-</a>
\ No newline at end of file
+    </header>
```
Comment (possumbilities) [inline docs/_includes/header.html:46]: there's a missing End of File here

**Comment by possumbilities** [comment]
@Sana-Maryam90 when running a visual comparison the typography doesn't match with creativecommons.org. I believe the issue is related to specificity scoping in the `body` rule within `style.css` that comes after the Vocabulary `@layer` import. 

Whatever occurs after that import overrides what's in the import. So the CSS is going to need to be reworked to sort that out. My suggestion would be removing the `body` rule and any other rules that clash with Vocabulary from style.css to sort this out. It might be that simple to fix.

**Comment by Sana-Maryam90** [comment]
@possumbilities I will try to fix it

**Pull request closed without merging**